hpOneView/resources/servers/server_hardware.py

```diff
@@ -43,118 +44,8 @@ class ServerHardware(object):
-
-                If the resource has no data, the UtilizationData is still returned, but will contain no samples and
-                sliceStartTime/sliceEndTime will be equal. oldestSampleTime/newestSampleTime will still be set
-                appropriately (null if no data is available). If the filter just does not happen to overlap the data
-                that a resource does have, then the metric history service will return null sample values for any
-                missing samples.
-
-            refresh:
-                Specifies that, if necessary, an additional request will be queued to obtain the most recent
-                utilization data from the iLO. The response will not include any refreshed data. To track the
-                availability of the newly collected data, monitor the TaskResource identified by the refreshTaskUri
-                property in the response. If null, no refresh was queued.
-
-            view:
-                Specifies the resolution interval length of the samples to be retrieved. This is reflected in the
-                resolution in the returned response. Utilization data is automatically purged to stay within storage
-                space constraints. See the following supported views.
-
-                native
-                    Resolution of the samples returned will be one sample for each 5-minute time period. This is the
-                    default view and matches the resolution of the data returned by the iLO. Samples at this resolution
-                    are retained up to one year.
-                hour
-                    Resolution of the samples returned will be one sample for each 60-minute time period. Samples are
-                    calculated by averaging the available 5-minute data samples that occurred within the hour, except
-                    for PeakPower, which is calculated by reporting the peak observed 5-minute sample value data during
-                    the hour. Samples at this resolution are retained up to three years.
-                day
-                    Resolution of the samples returned will be one sample for each 24-hour time period. One day is a
-                    24-hour period that starts at midnight GMT, regardless of the time zone in which the appliance or
-                    client is located. Samples are calculated by averaging the available 5-minute data samples that
-                    occurred during the day, except for PeakPower, which is calculated by reporting the peak observed
-                    5-minute sample value data during the day. Samples at this resolution are retained up to three
-                    years.
-
-        Returns:
-            dict
-        """
-
-        return self._client.get_utilization(id_or_uri, fields=fields, filter=filter, refresh=refresh, view=view)
-
-    def get_all(self, start=0, count=-1, filter='', sort=''):
```

Comment:
we may have to overwrite this method as done in logical enclosures to scopeUris as query param.

Reply:
There is no scopeUris param for server hardware.

Reply:
My Bad. Looks like I have mistakenly checked the POST endpoint where we have ScopeUris field.